_guides/quickstart-app-development.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Quickstart for DHIS2 App Development
+category: guide
+layout: page
+---
+
+# Quickstart for DHIS2 App Development
+
+## Create a new app and connect it to a local DHIS2 instance in 10 minutes or less.
+
+### Introduction
+You can start developing a DHIS2 app on OSX, Windows or Linux using Docker and the DHIS command line interface (CLI). In this guide, you will install the prerequisites, start up a local DHIS2 instance and create a new app. Your app will connect to the local DHIS2 instance.
+
+### Installing prerequisites
+If you are using Debian Linux, 
+1. Follow the [DHIS2 Docker guide](https://developers.dhis2.org/guides/dhis2-docker) to install the prerequisites
+
+If you are using OSX or Windows,
+1. Install [Docker](https://docs.docker.com/get-docker/)
+3. Install [yarn](https://classic.yarnpkg.com/en/docs/install)
+4. From the command line or terminal, install DHIS CLI
```

Comment:
on line 21, I'd add the link to the CLI:

`From the command line or terminal, install the DHIS [CLI](https://cli.dhis2.nu/#/getting-started){:target="\_blank"} globally`

Reply:
Good suggestion/

Pro GitHub tip: If you add "suggestion" after the first ``` in a code block, it can be committed by a user.  e.g. 👇

```suggestion
From the command line or terminal, install the DHIS [CLI](https://cli.dhis2.nu/#/getting-started){:target="\_blank"} globally
```